File: app/src/main/java/com/constraint/vagabond/main/details/DetailsContract.java
package com.constraint.vagabond.main.details;

import java.util.List;

public interface DetailsContract {

  interface Presenter {

    void loadImages(List<String> imageUrls);

    void loadAreaDescription(String recAreaName);

    void areaDirections(String recAreaName);
  }

  interface View {

    void setDataToRecyclerView(List<String> imageUrls);
  }
}

File: app/src/main/java/com/sierra/vagabond/main/details/DetailsContract.kt
package com.sierra.vagabond.main.details

interface DetailsContract {

    interface Presenter {

        fun loadImages(imageUrls: List<String>)

        fun loadAreaDescription(recAreaName: String)

        fun areaDirections(recAreaName: String)
    }

    interface View {

        fun setDataToRecyclerView(imageUrls: List<String>)

    }
}
